mypy/semanal.py

```diff
@@ -956,6 +956,10 @@ def get_all_bases_tvars(self, defn: ClassDef, removed: List[int]) -> TypeVarList
                 tvars.extend(base_tvars)
         return remove_dups(tvars)
 
+    def is_namedtuple_classdef(self, defn: ClassDef) -> bool:
+        base_exprs = defn.base_type_exprs
+        return any(getattr(b, 'fullname', None) == 'typing.NamedTuple' for b in base_exprs)
```

Comment:
This duplicates similar logic in `analyze_namedtuple_classdef`. There should be only one implementation of looking for a `typing.NamedTuple` base class.

Style nit: We avoid the use of `getattr` if there's a reasonable way around it, since it can't be type checked.